.github/workflows/JirabotMerge.yml

```diff
@@ -254,14 +254,17 @@ jobs:
                 print('Error: PROJECT_CONFIG is missing required fields: tagPrefix and/or tagPostfix')
                 sys.exit(1)
 
-            project_name = projectConfig.get('projectName')
-            if project_name is None:
-                print('Error: PROJECT_CONFIG is missing required field: projectName')
+            project_prefixes = projectConfig.get('projectPrefixes')
+            if project_prefixes is None:
```

Comment:
do we need to protect against empty projectPrefixes value?, not sure if this "None" test is only testing null.